repos/system_upgrade/common/libraries/grub.py

```diff
@@ -36,7 +36,7 @@ def blk_dev_from_partition(partition):
         api.current_logger().warning(
             'Could not get parent device of {} partition'.format(partition)
         )
-        raise StopActorExecution()
+        raise StopActorExecution()  # TODO: return some meaningful value/error
```

Comment:
:+1:  :heavy_plus_sign:  that's bug introduced basically when the private actor's library has been moved to shared library instead. Checking the library, it should be fixed everywhere, but let's keep it for another PR. Good catch!